README.md

```diff
@@ -102,6 +102,21 @@ git clone git@github.com:CMSgov/beneficiary-fhir-data.git ~/workspaces/bfd/benef
       -v 'bfd_pgdata:/var/lib/postgresql/data' \
       postgres:12 -c max_connections=200
     ```
+### Setup Local S3 using Minio Docker Container
+1. Set up a local S3 using Minio Docker Container
+    ```sh
+    docker run \
+  -p 9000:9000 \
+  -p 9001:9001 \
+  -e "MINIO_ROOT_USER=bfdLocalS3Dev" \
+  -e "MINIO_ROOT_PASSWORD=bfdLocalS3Dev" \
+  minio/minio server /data --console-address ":9001"
```

Comment:
just noticed that when I check the preview this is not formatting correctly. Also, to be consistent with postgres docker command I think it might be good to include the -d flag.

Reply:
Also adding this section here changes the flow of the native setup section which was previously numbered 1-7 and all steps using postgres. If we add this here I think it should not restart the numbering and we should also change any of the commands that follow in this section to use minio. Alternatively, this could be a new standalone section.

Reply:
done